mne_connectivity/utils/utils.py

```diff
@@ -53,20 +53,21 @@ def check_indices(indices):
 
     Parameters
     ----------
-    indices : tuple of array of int, shape (2, n_cons)
-        Tuple containing index pairs.
+    indices : tuple of length 2 of array_like, shape (n_cons)
```

Comment:
Nitpick, `shape` is always a tuple and `(n_cons)` is the same as `n_cons`, i.e., not a tuple. You need a trailing comma for a single-element tuple
```suggestion
    indices : tuple of length 2 of array_like, shape (n_cons,)
```

Reply:
Thanks for spotting! Have now fixed this and 2 other instances.